src/OrchardCore/Orchard.DisplayManagement/Theming/ThemeViewStart.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+
+namespace Orchard.DisplayManagement.Theming
+{
+    public class ThemeViewStart : Microsoft.AspNetCore.Mvc.Razor.RazorPage<dynamic>
+    {
+        public override Task ExecuteAsync()
+        {
+            Layout = "~/Views/Shared/_Layout.cshtml";
```

Comment:
Yes. it was the code from the original view. I prefer to keep it to be consistent,